integreat_cms/cms/templates/organizations/organization_list_row.html

```diff
@@ -31,6 +31,16 @@
             <i icon-name="x" class="text-red-500"></i>
         {% endif %}
     </td>
+    <td class="pr-2">
+        {% if organization.website %}
+            <a href="{{ organization.website }}"
+               title="{% translate "To website" %}">
+                <i icon-name="check" class="text-green-500"></i>
+            </a>
+        {% else %}
+            <i icon-name="x" class="text-red-500"></i>
+        {% endif %}
```

Comment:
```suggestion
        <a href="{{ organization.website }}"
           target="_blank"
           rel="noopener noreferrer"
           class="text-blue-500 hover:underline"
           title="{% translate "To website" %}">
            {{ organization.website }}
        </a>
```
I think we can directly show the website here.